graphql/src/query.rs

```diff
@@ -97,6 +97,19 @@ impl Query {
             .await?)
     }
 
+    pub async fn chapter(
```

Comment:
This should have a comment for what it's for and perhaps renaming it to chapter_contents would also be helpful.